src/App.js

```diff
@@ -33,21 +31,12 @@ export default function App() {
     }, []);
 
     // this will query the server API for the visitor history.
-    const getVisits = (limitTo) => {
-        fetch(
+    const getVisits = async (limitTo) => {
```

Comment:
you only need to use `async` if there is an `await` inside the function body, here it' not serving any purpose